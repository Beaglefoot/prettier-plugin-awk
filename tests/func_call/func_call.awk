{
  fn()
  fn(1,2+3)
  fn( a,   c )
  @ fn()
}
