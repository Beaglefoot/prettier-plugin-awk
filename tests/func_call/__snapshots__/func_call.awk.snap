{
  fn()
  fn(1,2+3)
  fn( a,   c )
  @ fn()
  very_long_function_name(first_argument, second_argument, third_argument, and_so_on, and_so_on_and_so_on)
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    fn()
    fn(1, 2 + 3)
    fn(a, c)
    @fn()
    very_long_function_name(\
        first_argument,
        second_argument,
        third_argument,
        and_so_on,
        and_so_on_and_so_on\
    )
}
